galley/pkg/runtime/projections/serviceentry/handler_test.go

```diff
@@ -0,0 +1,562 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serviceentry_test
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+
+	mcp "istio.io/api/mcp/v1alpha1"
+	networking "istio.io/api/networking/v1alpha3"
+	"istio.io/istio/galley/pkg/metadata"
+	"istio.io/istio/galley/pkg/runtime/processing"
+	"istio.io/istio/galley/pkg/runtime/projections/serviceentry"
+	"istio.io/istio/galley/pkg/runtime/projections/serviceentry/convert"
+	"istio.io/istio/galley/pkg/runtime/resource"
+	"istio.io/istio/pilot/pkg/model"
+
+	coreV1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/kubelet/apis"
+)
+
+const (
+	domainSuffix = "company.com"
+	clusterIP    = "10.0.0.1"
+	pod1IP       = "10.0.0.2"
+	pod2IP       = "10.0.0.2"
```

Comment:
Can you use a different value here? pod1IP is the same as pod2IP.

Reply:
done.